src/models/imposter.js

```diff
@@ -54,6 +54,9 @@ function create (Protocol, request) {
 
                 result.requests = server.requests;
                 result.stubs = server.stubs();
+                result.stubs.forEach(function (stub) {
```

Comment:
Got it! Makes perfect sense.
I've made the suggested changes and pushed a new commit. Let me know if there is anything I've missed.
Thanks.